ci/travis/build-autoscaler-images.sh

```diff
@@ -11,12 +11,13 @@ DOCKER_USERNAME="raytravisbot"
 if [[ "$TRAVIS" == "true" && "$TRAVIS_PULL_REQUEST" == "false" ]]; then
     echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 
-    docker build -q -t rayproject/base-deps docker/base-deps
-
     wheel=$(cd $ROOT_DIR/.whl; ls | grep cp36m-manylinux)
     commit_sha=$(echo $TRAVIS_COMMIT | head -c 6)
     cp -r $ROOT_DIR/.whl $ROOT_DIR/docker/autoscaler/.whl
 
+    docker build -q -t rayproject/base-deps:$commit_sha docker/base-deps
```

Comment:
You need to do this in two steps because `docker build ayproject/autoscaler` requires `rayproject/base-deps` without the trailing commit sha. So we need in psuedocode (plz check):
```
docker build -t rayproject/base-deps ...
docker tag rayproject/base-deps rayproject/base-deps:$sha
docker push rayproject/base-deps:$sha
```

Reply:
Oh! Yes let me fix that1